Sources/HTTPParser/http_parser.swift

```diff
@@ -254,17 +390,18 @@ public enum http_errno: String, Error {
   }
 }
 
-public class http_parser {
+private func LIKELY(_ X: Bool) -> Bool { return X }
```

Comment:
I would `@inline(__always)` such until the compiler crashes :-)

          (__)
        /  .\/.     ______
       |  /\_|     |      \
       |  |___     |       |
       |   ---@    |_______|
    *  |  |   ----   |    |
     \ |  |_____
      \|________|
CompuCow Discovers Bug in Compiler